tests/scripts/scriptlib.fsx

```diff
@@ -10,6 +10,14 @@ open System.IO
 open System.Text
 open System.Diagnostics
 
+module MessageSink =
+    let sinkWriter =
+#if DEBUG
+        Console.Out
```

Comment:
Why DEBUG only?
Won't this bring differences in results across Debug/Release then?

Reply:
It affects only the diagnostic output, this is not related to the actual test cases output. This just turns off the flood of diagnostic messages printed by `log` function during the command line test run.

The general idea is to direct all the `log` calls to the unredirected console via this `sinkWriter` TextWriter.
The `log` calls originate mostly from our test infrastructure, there's plenty of them and they currently show up during the commandline test run.

OTOH everything that the test cases print to stdout and stderr can be captured and inspected per test case. With this PR it does not show up during command line run.

Reply:
To protect CLI output, got it - good in that case.